src/main/flight/pid.c

```diff
@@ -130,6 +130,91 @@ float getdT (void) {
     return dT;
 }
 
+// calculates strength of horizon leveling; 0 = none, 100 = most leveling
+int calcHorizonLevelStrength(uint16_t rxConfigMidrc, int horizonInclFact,
+                                                          int horizonSensit)
+{
+    int horizonLevelStrength, inclinationLevelRatio, factorRatio;
```

Comment:
Can you declare variables at first point of usage. This makes the code easier to read and more maintainable.